retrieveProducts/src/handlers/getProductById.js

```diff
@@ -0,0 +1,34 @@
+import { loadProductById } from '../service';
+import { buildResponse } from '../utils';
+
+const getIdFromRequest = (event) => {
+  try {
+    console.log(`Incoming request ${JSON.stringify(event)}`);
+    const { id } = event.pathParameters;
```

Comment:
Don't you need certain path parameter?

Reply:
What do you mean by `certain`. In this case I use path parameter with name `id`. If you check api gateway configuration you will see the same path parameter `path: products/{id} `

Reply:
Sorry, my bad. Everything is correct indeed